app/helpers/data.py

```diff
@@ -204,20 +204,36 @@ def add_session_to_event(request, event_id, state=None, use_current_user=True, n
         """
         form = request.form
         slide_temp_url = form.get('slides_url')
+        slide_custom_url= form.get('slides_custom_url')
+
         video_temp_url = form.get('video_url')
+        video_custom_url= form.get('video_custom_url')
+
         audio_temp_url = form.get('audio_url')
+        audio_custom_url= form.get('audio_custom_url')
+
```

Comment:
I think these extra fields are not needed, this can be implemented using `slides_url`, etc. itself.